src/land/ascat_ssm2ioda.py

```diff
@@ -62,8 +62,7 @@ def _read(self):
         # open input file name
         ncd = nc.Dataset(self.filename, 'r')
         # set and get global attributes
-        AttrData["satellite"] = "METOP"
-        AttrData['sensor'] = ncd.getncattr('instrument_name')
```

Comment:
instrument name is ASCAT.  As satellite will show METOP-A ASCAT or METOP-B ASCAT  or METOP-C ASCAT , I removed Instrument name ASCAT to avoid the duplication.

Reply:
Avoid the redundancy when added instrument name.